README.md

```diff
@@ -73,6 +65,15 @@ The following options are also supported and will be passed directly to the
 - [`lastModified`](https://www.npmjs.com/package/send#lastmodified)
 - [`maxAge`](https://www.npmjs.com/package/send#maxage)
 
+### Handling Errors
+
+If an error occurs while trying to send a file, the error will be passed
+to Fastify's error handler. You can set a custom error handler with
+[`fastify.setErrorHandler()`](https://www.fastify.io/docs/latest/Server-Methods/#seterrorhandler).
+
+If a request matches the URL `prefix` but a file cannot be found for the request,
+a `404 Not Found` error will be created and passed to Fastify's error handler.
```

Comment:
I think this behavior might be wrong. We should really flow this through the notFoundHandler.

Reply:
Yes that would be better. From looking at https://github.com/fastify/fastify/blob/master/fastify.js I think one way to do that would be for `fastify-static` to have access to Fastify's [`fourOhFour` router](https://github.com/fastify/fastify/blob/master/fastify.js#L149) so that it can call `fourOhFour.lookup(req, res)`. Do you agree?

Reply:
I recal (but I might be wrong, I’m from a phone) that Every plugin has its own 404 handler defined, so we should just use that.

Reply:
Yes you're right. I figured out how to call the 404 handler and I've updated the PR :)